tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def reference_path():  # reference EXE_DIR in zenodo
+    return "/home/surface10/mrasolon/files_for_zenodo/reference/EXE_DIR/"
```

Comment:
Is there a way to pass this as a parameter, or have it as some pytest config file?

Reply:
Yeah I can add the config path as an argument, and return the reference data path in the config